src/components/NavBar/NavMenu.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import AccountCircle from '@mui/icons-material/AccountCircle';
+import IconButton from '@mui/material/IconButton';
+import Link from '@mui/material/Link';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import { useSession } from '@inrupt/solid-ui-react';
+
```

Comment:
I think we can add JSDoc comments here for the NavMenu component like what we have for NavBar